app/models/user.rb

```diff
@@ -53,4 +50,15 @@ def update_password_and_token(password)
     self.token = nil
     self.save
   end
+
+  def connect_with_invitors
+    invites = Invite.where(email: self.email)
+    if invites.exists?
+      invitor_ids = invites.map(&:user_id).uniq
```

Comment:
It costs very little to implement, so I think it's a great idea! It's also interesting how `obj.pluck.uniq` is faster because it is interpreted differently from `obj.uniq.pluck`